src/genie/libs/parser/iosxe/tests/ShowLispEidTableVrfUserIpv4MapCache/cli/equal/golden_output_expected.py

```diff
@@ -0,0 +1,571 @@
+expected_output = {
+    "vrf": "User",
+    "iid": 4100,
+    "number_of_entries": 2186,
```

Comment:
let's have `vrf` as level.
```suggestion
expected_output = {
    "vrf": {
        "User": {
            "iid: 4100,
            "number_of_entries": 2186,
```

Reply:
Updated